Add explicit criteria for merging PRs (#2116)
projectblacklight/blacklight: CONTRIBUTING.md

* After making your changes, be sure to run the [Blacklight tests](https://github.com/projectblacklight/blacklight/wiki/testing) to make sure everything works.
* Submit your change as a [Pull Request](https://help.github.com/en/articles/about-pull-requests).

## Pull Requests and Code Review

> - Pull requests must be reviewed and approved by one non-author with write-permissions

Reviewer: ```suggestion
- Pull requests must be reviewed and approved by someone who isn't the author
```

Author: I'd prefer to keep this language. See: https://github.com/projectblacklight/blacklight/pull/2116#issuecomment-506525886

Reviewer: Works for me, I do appreciate the clarity. 👍

- A reviewer who has approved a PR but has not merged it should provide a comment (e.g. "This looks good, I'm just waiting for Travis to build", "This is a breaking change. I would like at least one more reviewer.")

> - An approved PR may be merged by the reviewer, the author, or another committer

Reviewer 2: My preference is to codify the current working practice as I understand over the past 6 years I have been involved with process. If a reviewer is to approve the pull request, I hope that they could take ownership over that process or ask for more assistance to see it through.

```suggestion
- An approved PR may be merged by the reviewer or another committer
```

Reviewer 2: Not a blocking comment/suggestion just an opinion ^^

Reviewer 2: I'm going to remove this suggestion for now in favor moving @camillevilla's PR forward. Hopefully this can be discussed more.
